### ISSUE TITLE
Fails to Load YTM On application start up. (ERR_FAILED (-2))

### ISSUE THREAD

**Issue by cfsmp3** [ytmdesktop/ytmdesktop#1004]
### Consent

- [X] I verified that there is no open/closed issue for the same subject.
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Current Behavior

From a freshly downloaded AppImage, after removing configuration files (~/.config/youtube etc):

```
✦ ❯ ./YouTube\ Music\ Desktop\ App-1.14.2.AppImage 
[!] To add translation for your introduced feature, you may navigate to 
[!] /home/cfsmp3/.config/youtube-music-desktop-app/locales to modify the your locale file.
[!] Then kindly open a PR to ytmdesktop-locales repo. :)
Checking for update
Generated new staging user ID: ccde987d-8771-5253-9986-336f9bf547e9
(node:18037) UnhandledPromiseRejectionWarning: Error: ERR_FAILED (-2) loading 'https://music.youtube.com'
    at rejectAndCleanup (electron/js2c/browser_init.js:205:1493)
    at Object.stopLoadingListener (electron/js2c/browser_init.js:205:1868)
    at Object.emit (events.js:327:22)
(node:18037) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:18037) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
Update for version 1.14.2 is not available (latest version: 1.13.0, downgrade is disallowed).
```

And nothing happens - just a spinning wheel.

### Expected Behavior

I'd expect the usual app to work

### Steps To Reproduce

See terminal above, it's just loading the app. I don't need to go out of my way to trigger it.

### YTMDesktop

v1.14.2

### OS

Ubuntu

### OS Version

22.04

### Arch

x64

### Installation way

AppImage

**Comment by cfsmp3** [comment]
> Can I confirm that this issue is still not happening? This appears to simply be an issue of being unable to Load YTM. thinking
> 
> Looking into this issue, there's been no solid way of solving it on GitHub and Stackoverflow: [electron/electron#18857](https://github.com/electron/electron/issues/18857)

I am still getting it. Noteworthy: This started after updating from Ubuntu 21.10 to 22.04, which of course updated a bunch of things. This is the only thing that is broken though.

**Comment by sfllaw** [comment]
I worked around this by starting the AppImage with `--no-sandbox`.

**Comment by cfsmp3** [comment]
> I worked around this by starting the AppImage with `--no-sandbox`.

That gets me to the next stage at least, thanks :-)
Now it's the "this browser might not be safe" problem.

**Comment by Alipoodle** [comment]
is that issue which has a few "temporary" fixes #866 
But I'm still confused to why Sandboxing is causing it to fail to load 🤔
_I don't truly know what Sandboxing is doing here_

**Comment by Alipoodle** [comment]
Closing issue due to no longer supporting YouTube Music Desktop v1 + Release of YouTube Music v2.
Please re-open this issue if this is still occurring in the latest build of the application, however as a re-write has been done on the entire project most issues will be fixed, no longer a feature or re-worked.
